Linux/flash

```diff
@@ -24,6 +24,7 @@ OPTIONS:
   --help|-h       Show this message
   --bootconf|-C   Copy this config file to /boot/config.txt
   --config|-c     Copy this config file to /boot/device-init.yaml (or occidentalis.txt)
+  --onlyconf|-o   Configure an already written image (useful for testing)
```

Comment:
I wonder if we could do the "onlyconf" by omitting the download link/file name.
`-o` is often used as output parameter. What about `-u` for "update conf only" ?

Reply:
I like your suggestion of getting into configuration mode where no image is provided. I will update the pull request.
Thanks!